src/ddspdrum/module.py

```diff
@@ -268,10 +245,46 @@ def __call__(self, envelopecontrol: np.array, audiosample: np.array):
         envelopecontrol = np.clip(envelopecontrol, 0, 1)
         audiosample = np.clip(audiosample, -1, 1)
         amp = self.control_to_sample_rate(envelopecontrol)
-        signal = self.fix_length(audiosample, len(amp))
+        signal = fix_length(audiosample, len(amp))
         return amp * signal
 
 
+class Drum:
```

Comment:
will fold the filters into this eventually when I figure out the differentiability aspect of them.